apps/front/src/components/user/UserUpdateForm.vue

```diff
@@ -0,0 +1,60 @@
+<template>
```

Comment:
Same feedbacks than UserCreateForm

Also about naming CreateForm should be sufficient and play well with nuxt's component auto-loading